media-video/droidcam/droidcam-1.4.ebuild

```diff
@@ -0,0 +1,106 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit readme.gentoo-r1 desktop linux-mod xdg-utils
+
+DESCRIPTION="Turn your mobile device into a webcam"
+HOMEPAGE="https://www.dev47apps.com/"
+SRC_URI="https://github.com/aramg/droidcam/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="gtk"
+RESTRICT="bindist mirror"
+
+DEPEND="gtk? (
+				dev-cpp/gtkmm:3.0
+				media-video/ffmpeg
+			)
+		!gtk? ( media-video/ffmpeg[-X] )
```

Comment:
Why is it required that ffmpeg be installed without the X flag if gtk is not set?

Reply:
Indeed, I was testing/experimenting and forgot that one in.